python/feather/interop.h

```diff
@@ -667,6 +668,55 @@ class FeatherDeserializer {
   PyObject* out_;
 };
 
+PyObject* primitive_mask(const PrimitiveArray& arr) {
+  npy_intp dims[1] = {static_cast<npy_intp>(arr.length)};
+  PyObject* out = PyArray_SimpleNew(1, dims, NPY_BOOL);
+  if (out == NULL) return out;
+
+  uint8_t* out_values = reinterpret_cast<uint8_t*>(PyArray_DATA(out));
+  if (arr.null_count > 0) {
+    for (int64_t i = 0; i < arr.length; ++i) {
+      out_values[i] = util::get_bit(arr.nulls, i) ? 1 : 0;
+    }
+  } else {
+    for (int64_t i = 0; i < arr.length; ++i) {
+      out_values[i] = 1;
+    }
+  }
+  return out;
```

Comment:
If you are only using this to set nulls, why not set 1 for null, 0 for not-null, and skip the invert steps (extra computation cost)?